src/Morphir/Visual/Components/DecisionTree.elm

```diff
@@ -30,88 +30,60 @@ type alias BranchNode =
     }
 
 
-highlightColor : { true : Color, false : Color, default : Color }
+highlightColor : { highlighted : Color, default : Color }
 highlightColor =
-    { true = Color 100 180 100
-    , false = Color 180 100 100
-    , default = Color 120 120 120
+    { highlighted = rgb255 0 163 255
+    , default = rgb255 120 120 120
     }
 
 
-type Color
-    = Color Int Int Int
-
-
-type HighlightState
-    = Highlighted Bool
-    | NotHighlighted
-
-
 type Direction
     = Horizontal
     | Vertical
 
 
-highlightStateToColor : HighlightState -> Color
-highlightStateToColor state =
-    case state of
-        Highlighted bool ->
-            if bool then
-                highlightColor.true
-
-            else
-                highlightColor.false
-
-        NotHighlighted ->
-            highlightColor.default
-
-
-highlightStateToBackground : HighlightState -> String
-highlightStateToBackground state =
-    case state of
-        NotHighlighted ->
-            "repeating-linear-gradient( 45deg, #565656, #9a9595 10px, #656565 10px, #797979 20px )"
-
-        _ ->
-            "none"
-
+highlightStateToColor : Bool -> Color
+highlightStateToColor highlighted =
+    if highlighted then
+        highlightColor.highlighted
 
-highlightStateToBorderWidth : HighlightState -> Int
-highlightStateToBorderWidth state =
-    case state of
-        Highlighted _ ->
-            4
+    else
+        highlightColor.default
 
-        NotHighlighted ->
-            2
 
+highlightStateToBorderWidth : Bool -> Int
+highlightStateToBorderWidth highlighted =
+    if highlighted then
+        4
 
-highlightStateToFontWeight : HighlightState -> Attribute msg
-highlightStateToFontWeight state =
-    case state of
-        Highlighted _ ->
-            Font.bold
+    else
+        2
 
-        NotHighlighted ->
-            Font.regular
 
+highlightStateToFontWeight : Bool -> Attribute msg
+highlightStateToFontWeight highlighted =
+    if highlighted then
+        Font.bold
 
-toElementColor : Color -> Element.Color
-toElementColor (Color r g b) =
-    rgb255 r g b
+    else
+        Font.regular
 
 
 toCssColor : Color -> String
-toCssColor (Color r g b) =
-    String.concat [ "rgb(", String.fromInt r, ",", String.fromInt g, ",", String.fromInt b, ")" ]
+toCssColor color =
+    let
+        rgb =
+            toRgb color
+    in
+    String.concat [ "rgb(", String.fromFloat (rgb.red * 255), ",", String.fromFloat (rgb.green * 255), ",", String.fromFloat (rgb.blue * 255), ")" ]
```

Comment:
What's the purpose of this change? Was it translating colors incorrectly before?

Reply:
It looks weird on it's own, but with the other changes you commented on you can see that we were using a custom `Color Int Int Int` type, instead of `Element.Color`. You have to call toRgb on the latter to get the color values back